src/molecules/tables/index.tsx

```diff
@@ -236,32 +241,38 @@ export default function DataTable<TData, TValue>(
           </TableHeader>
           <TableBody className={cn(classNames?.table?.body)}>
             {table.getRowModel().rows?.length ? (
-              table.getRowModel().rows.map((row) => (
-                <Fragment key={row.id}>
-                  <TableRow
-                    data-state={row.getIsSelected() ? 'selected' : undefined}
-                    className="whitespace-nowrap"
-                  >
-                    {row.getVisibleCells().map((cell) => (
-                      <TableCell key={cell.id}>
-                        {
-                          flexRender(
-                            cell.column.columnDef.cell,
-                            cell.getContext()
-                          ) as JSX.Element
-                        }
-                      </TableCell>
-                    ))}
-                  </TableRow>
-                  {row.getIsExpanded() && renderSubComponent && (
-                    <TableRow>
-                      <TableCell colSpan={row.getVisibleCells().length}>
-                        {renderSubComponent({ row })}
-                      </TableCell>
+              table
+                .getRowModel()
+                .rows.slice(
+                  pagination.pageIndex * pagination.pageSize,
+                  (pagination.pageIndex + 1) * pagination.pageSize
+                )
+                .map((row) => (
```

Comment:
To display only the rows of the selected page, previously, all rows were shown regardless of the page number or rows per page count.

Reply:
Thanks for the comment, I just double checked the code, this issue is happening since we are setting ManualPagination to true in the code. If we set it to false we do not need such code. I am going to share few comments about the code and can accept it after it's being applied.